### PR TITLE
Quote logic

### PR DESCRIPTION
**Pull request by freedannyv**
# Issue Being Addressed

#25 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[x] Update to Existing Feature
[ ] Other (state below)

# Description

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

Updated the logic to show only 1 quote per time, looping through the players so that each time a different player is quoted. Plus, ensured all Hole In One quotes are displayed too. I created a **counter** in the Vuex state to increment by 1 each time the Current-Total page is created, with the sufficient actions, mutations and getters.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
2. Go to 'current-total'
3. Click on: refresh the page or click through to create new scores for the next hole to see the results of the quotes.
4. See the updated element x

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Suggestion: make hole in one quotes more interesting with different styling

### CODE REVIEW

**Review by paulchrisluke — APPROVED** [review]
ux approved pending code review to merge from @giteshk11

File: src/store/modules/gameInfo.js [hunk 1]
```diff
@@ -34,17 +38,30 @@ const actions = {
...
     state.gameInfo = payload;
+  },
+  increaseCounter(state) {
+    if (state.counter < state.gameInfo.playersInfo.length - 1) {
+      console.log(state.counter);
+      return state.counter++;
+    } else {
+      console.log(state.counter);
+      return (state.counter = 0);
+    }
```
Comment (giteshk11) [inline src/store/modules/gameInfo.js:64]: This logic should be performed in actions rather than in mutations.

File: src/views/CurrentTotal.vue [hunk 2]
```diff
@@ -70,20 +69,20 @@ export default {
   data() {
     return {
       playersInfo: [],
-      counter: 1,
       par: 4,
       quote
     };
   },
   created() {
+    this.increaseCounter();
```
Comment (giteshk11) [inline src/views/CurrentTotal.vue:77]: call this function inside `then()` after server call

**Review by giteshk11 — CHANGES_REQUESTED** [review]

File: src/store/modules/gameInfo.js (hunk 1 above)
Comment (freedannyv) [reply src/store/modules/gameInfo.js]: Updated

File: src/views/CurrentTotal.vue (hunk 2 above)
Comment (freedannyv) [reply src/views/CurrentTotal.vue]: Updated

**Review by giteshk11 — APPROVED** [review]

**Pull request merged**